lib/metasploit/framework/password_crackers/cracker.rb

```diff
@@ -343,46 +343,46 @@ def cracker_version
         # @return [Array] An array set up for {::IO.popen} to use
         def john_crack_command
           cmd_string = binary_path
-          cmd = [ cmd_string,  '--session=' + cracker_session_id, '--nolog' ]
+          cmd = [cmd_string,  '--session=' + cracker_session_id, '--nolog']
```

Comment:
You'll probably want to interpolate strings in the future for readability.

Reply:
I noticed and thought about that. Those are mainly left over from the old John implementation. Decided to not fix it since it wasn't broken at this go around